doc/source/guide/merging.rst

```diff
@@ -25,3 +25,11 @@ duplicating :option:`-a/--add-tracefile<gcovr --add-tracefile>`. With this optio
 you have to place your pathnames with wildcards in double quotation marks::
 
     gcovr --add-tracefile "run-*.json" --html-details coverage.html
+
+If you want to merge coverage reports generated in different `--root` directories you
+can use the option `--json-base=dir/to/prepend` to get the same root directory for all
```

Comment:
```suggestion
can use the option ``--json-base=dir/to/prepend`` to get the same root directory for all
```

RST is a very “special” markup format. Single delimiters like `` `text` `` or `'text'` are just *italics*. For code markup, there must be two backticks.

Alternatively, we could place a link to the ``:option:`--json-base-dir` `` here.